flycheck.el

```diff
@@ -8120,8 +8120,9 @@ See URL `https://github.com/CSSLint/csslint'."
 (defconst flycheck-stylelint-args '("--formatter" "json")
   "Common arguments to stylelint invocations.")
 
-(flycheck-def-config-file-var flycheck-stylelintrc
-    (css-stylelint scss-stylelint sass-stylelint less-stylelint) nil)
+(let ((print-length 3))
```

Comment:
Might be good at some comment explaining the need for this here, as without the PR context it won't be obvious.

Reply:
Good point. I added a comment.